common/djangoapps/enrollment/tests/test_views.py

```diff
@@ -1438,38 +1440,62 @@ def setUp(self):
             email=self.EMAIL,
             password=self.PASSWORD,
         )
+        self.staff_user = UserFactory.create(
+            username=self.STAFF_USERNAME,
+            email=self.STAFF_EMAIL,
+            password=self.PASSWORD,
+            is_staff=True,
+        )
+        self.logged_in_user = self.user
```

Comment:
I think putting this as global variable is confusing. Here is what I think would be better:
have function `self._login(is_staff)` return the actual user logged in.  That way, you don't need to manage a global variable.

Reply:
👍